robottelo/ui/settings.py

```diff
@@ -13,6 +13,27 @@ def navigate_to_entity(self):
         """Navigate to Settings entity page"""
         Navigator(self.browser).go_to_settings()
 
+    def remove_parameter(self, tab_locator, param_name):
+        """Removes the value of selected parameter under settings"""
+        if self.wait_until_element(tab_locator) is None:
+            raise UINoSuchElementError(
+                "Couldn't find the tab with name: '%s'" % tab_locator)
+        self.click(tab_locator)
+
+        loc = locators['settings.edit_param']
+        if self.wait_until_element(loc % param_name) is None:
+            raise UINoSuchElementError(
+                'Could not find edit button to update selected param'
+            )
+        else:
+            self.click(loc % param_name)
+            # if parameter is already empty
+            if self.wait_until_element(locators['settings.remove']) is None:
```

Comment:
I would suggest to think about decreasing timeout for that waiter like 
`self.wait_until_element(locators['settings.remove'], timeout=3) is None`
as it is unacceptable to wait for 12 seconds for each empty parameter